docs/reference_guide.md

```diff
@@ -509,9 +509,9 @@ Examples in situ:
 
 ### 10. map.increment()
 
-Syntax: ```map.increment(&key)```
+Syntax: ```map.increment(key)```
 
-Increments the key's value by one. Used for histograms.
+Increments the key's value by one. Used for histograms. This is actually translated to `lookup()` and `update()` calls.
```

Comment:
Hm, you're right. It's confusing and not coherent with the rest of the doc, which doesn't give such details. I'll remove.